pyramid_oereb/core/renderer/getegrid/templates/xml/getegrid.xml

```diff
@@ -1,4 +1,6 @@
-## -*- coding: utf-8 -*-
+<%
+    from pyramid_oereb import Config
+%>
 <?xml version="1.0" encoding="UTF-8" ?>
 <extract:GetEGRIDResponse
                         xmlns:xsd="http://www.w3.org/2001/XMLSchema"
```

Comment:
`xmlns:data="http://schemas.geo.admin.ch/V_D/OeREB/2.0/ExtractData"` missing?

Reply:
no-this was copy paste error

Reply:
Are you sure? VSC reports that `<data:Code>` code is not bound.